CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [1.19.1] - 2021-01-14
+
+### Features
+
+- Add an optional parameter `captchaToken` for captcha support in loginWithPassword
```

Comment:
Don't forget to update the documentation of the method (you can even add a link to the documentation in the changelog).